Amplitude.xcodeproj/xcshareddata/xcschemes/Amplitude_iOSTests.xcscheme

```diff
@@ -22,6 +22,16 @@
                ReferencedContainer = "container:Amplitude.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
```

Comment:
Adding a new test schema? Was this during the try or intentional? I guess it won't hurt though, just if we have a new schema for test do we also want to config it in github action to test through?

Reply:
Not intentional. Must have been one of my test schemas.